### PR TITLE
[translation] Russian

### PR DESCRIPTION
**Pull request by DmitryDrozdik**

### CODE REVIEW

File: config/translations/ru/database.dump.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-description: 'Дамп структуры и контента MySQL БД и таблиц'
+description: 'Дамп структуры и содержимого базы данных'
```
Comment (enzolutions) [inline config/translations/ru/database.dump.yml]: Please don't translate MySQL

File: config/translations/ru/database.restore.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-description: 'Восстановление структуры и контента MySQL БД и таблиц'
+description: 'Восстановление структуры и содержимого базы данных'
```
Comment (enzolutions) [inline config/translations/ru/database.restore.yml]: Please don't translate mysql

File: config/translations/ru/database.dump.yml (hunk 1 above)
Comment (DmitryDrozdik) [inline config/translations/ru/database.dump.yml]: @enzolutions that translation was wrong, there is not mention about MySQL in the https://github.com/hechoendrupal/DrupalConsole/blob/master/config/translations/en/database.dump.yml

File: config/translations/ru/database.restore.yml (hunk 2 above)
Comment (DmitryDrozdik) [inline config/translations/ru/database.restore.yml]: @enzolutions same here https://github.com/hechoendrupal/DrupalConsole/blob/master/config/translations/en/database.restore.yml

**Comment by enzolutions** [comment]
Thanks @DmitryDrozdik

**Pull request merged**